src/node/txreconciliation.cpp

```diff
@@ -39,7 +39,8 @@ class TxReconciliationState
      * the following commits.
      *
      * Reconciliation protocol assumes using one role consistently: either a reconciliation
-     * initiator (requesting sketches), or responder (sending sketches). This defines our role.
+     * initiator (requesting sketches), or responder (sending sketches). This defines our role,
+     * based on the direction of the p2p connection.
```

Comment:
Note you have a bunch of references to the term initiator left in `reconciatliation.{h,cpp}`:

```
src/node/txreconciliation.cpp:     * initiator (requesting sketches), or responder (sending sketches). This defines our role,
src/node/txreconciliation.h: * 2.  At regular intervals, a txreconciliation initiator requests a sketch from a peer, where a
src/node/txreconciliation.h: * 3.  Once the initiator received a sketch from the peer, the initiator computes a local sketch,
src/node/txreconciliation.h: * 4b. If the difference was larger than estimated, initial txreconciliation fails. The initiator
src/node/txreconciliation.h: * SUCCESS. The initiator knows full symmetrical difference and can request what the initiator is
src/node/txreconciliation.h: * FAILURE. The initiator notifies the peer about the failure and announces all transactions from
```
It's minor, I think they can be addressed in its own commit here or in #26283 to avoid another review round.

Reply:
I think it makes sense to keep using the more abstract terms initiator / receiver within the txreconciliation module, even after removing it from the `sendtxrcncl` message.